android/src/main/java/com/pdftron/reactnative/views/DocumentView.java

```diff
@@ -1949,6 +1951,17 @@ public void onHostFragmentViewCreated() {
                     }
                 });
             }
+        } else if (mPdfViewCtrlTabHostFragment instanceof RNCollabViewerTabHostFragment) {
```

Comment:
missing check of `mPdfViewCtrlTabHostFragment != null && mPdfViewCtrlTabHostFragment.getView() == null`, we don't want to double-add